cmd/cli/plugin/secret/README.md

```diff
@@ -0,0 +1,173 @@
+     private-repo-fetch-0          registry.pivotal.io  not exported       15d  test-ns
+     antrea-fetch-0                registry.pivotal.io  not exported       15d  tkg-system
+     metrics-server-fetch-0        registry.pivotal.io  not exported       15d  tkg-system
+     tanzu-addons-manager-fetch-0  registry.pivotal.io  not exported       15d  tkg-system
+     tanzu-core-fetch-0            registry.pivotal.io  not exported       15d  tkg-system
+   ```
+
+##### List registry secrets in json output format
+
+   ```sh
+   >>> tanzu secret registry list -n kapp-controller-packaging-global -o json
+   [
+     {
+       "age": "15d",
+       "exported": "to all namespaces",
+       "name": "pkg-dev-reg",
+       "registry": "us-east4-docker.pkg.dev"
+     }
+   ]
+   ```
+
+##### List registry secrets in yaml output format
+
+   ```sh
+   >>> tanzu secret registry list -n kapp-controller-packaging-global -o yaml
+   - age: 15d
+     exported: to all namespaces
+     name: pkg-dev-reg
+     registry: us-east4-docker.pkg.dev
+   ```
+
+##### List registry secrets in table output format
+
+   ```sh
+   >>> tanzu secret registry list -n kapp-controller-packaging-global -o table
+   / Retrieving registry secrets...
+     NAME         REGISTRY                 EXPORTED           AGE
+     pkg-dev-reg  us-east4-docker.pkg.dev  to all namespaces  15d
+   ```
+
+#### Delete a registry secret
+
+   The "delete" command deletes a v1/Secret of type kubernetes.io/dockerconfigjson from the specified namespace. If no namespace is specified, the secret will be deleted from the default namespace (if existing).
+   In case a SecretExport resource with the same name exists, it will be deleted from the namespace as well.
+
+   ```sh
+   >>> tanzu secret registry delete test-secret -n test-ns
+   Deleting registry secret 'test-secret' from namespace 'test-ns'. Are you sure? [y/N]: y
+   \ Deleting registry secret 'test-secret'...
+     Deleted registry secret 'test-secret' from namespace 'test-ns'
+   ```
+
+#### Update a registry secret
+
+   The "update" command updates the v1/Secret of type kubernetes.io/dockerconfigjson.
+   In case of specifying the --export-to-all-namespaces flag, the SecretExport resource will also get updated. Otherwise, there will be no changes in the SecretExport resource.
+
+##### Update a registry secret. There will be no changes in the associated SecretExport resource
+
+   ```sh
+   >>> tanzu imagepullsecret update test-secret --username test-user -n test-ns --password-env-var PASSENV
+   \ Updating image pull secret 'test-secret'...
+     Updated image pull secret 'test-secret' in namespace 'test-ns'
+   ```
+
+##### Update a registry secret with 'export-to-all-namespaces' flag being set
+
+   ```sh
+   >>> tanzu imagepullsecret update test-secret--username test-user -n test-ns --password-env-var PASSENV --export-to-all-namespaces=true
```

Comment:
Same as above.
Need to change imagepullsecret => secret registry
Same with output for the command